process/project-lifecycle.md

```diff
@@ -0,0 +1,161 @@
+# Project Life Cycle
+
+Projects are either directly under the governance of the Technical Advisory Committee (TAC) or a specific Working Group (WG). The TAC is responsible for reviewing and approving all significant changes to a Project's life cycle. Where a Project reports into a specific WG, that WG can support the Project's progression and provide recommendations to the TAC.
```

Comment:
The second sentence here could be read as contradicting the first. Is the intended workflow that all net new projects receive TAC approval, and then after that approval, some may report up to WGs instead of the TAC if there is a relevant WG?

Reply:
No, the intent is that whether projects report to the TAC or a WG their lifecycle is under the control of the TAC (I think this is however not a settled issue). I'm clarifying the text though. Thanks.